README.md

```diff
@@ -48,3 +101,47 @@ See Bazel core [documentation](https://docs.bazel.build/versions/master/be/pytho
 ## py_binary
 
 See Bazel core [documentation](https://docs.bazel.build/versions/master/be/python.html#py_binary).
+
+<a name="py_test"></a>
+## py_test
+
+See Bazel core [documentation](https://docs.bazel.build/versions/master/be/python.html#py_test).
+
+<a name="pip_import"></a>
+## pip_import
```

Comment:
There's a thing called [skydoc](https://github.com/bazelbuild/skydoc) that will turn structured docstrings into Bazel documentation.  It's kind of basic and a bit, well, not beautiful.  https://google.github.io/subpar/subpar.html

Reply:
I'll TAL at skydoc for this repo, but in `rules_docker` and `rules_k8s` I'm blocked on [this](https://github.com/bazelbuild/rules_docker/issues/32#issuecomment-324164587)

Reply:
Ok, I've switched over to skydoc for the docs here (based on how google/subpar does it).  Feedback welcome :)